public/controllers/dev-tools/dev-tools.js

```diff
@@ -248,6 +279,69 @@ export class DevToolsController {
     this.groups = this.analyzeGroups();
     const currentGroup = this.calculateWhichGroup();
     this.highlightGroup(currentGroup);
+
+    // Register our custom Codemirror hint plugin.
+    CodeMirror.registerHelper('hint', 'dictionaryHint', function (editor) {
+      const model = editor.model;
+      function getDictionary(line, word) {
+        let hints = {};
```

Comment:
- You are initializing it as an `object`, then it's used as an array.
- You are initializing it as `let`, IMO it would be `const`.